kube-config-files/methode-image-model-mapper-service.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: methode-image-model-mapper
+  labels:
+    app: methode-image-model-mapper
+    visualize: "true"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: methode-image-model-mapper
+  template:
+    metadata:
+      labels:
+        app: methode-image-model-mapper
+        visualize: "true"
+    spec:
+      containers:
+      - name: methode-image-model-mapper
+        image: coco/methode-image-model-mapper:v16.0.2-rc.1
+        env:
+        - name: JAVA_OPTS
+          value: "-Xms256m -Xmx256m -XX:+UseG1GC -server"
+        - name: KAFKA_PROXY
+          value: "kafka:8082"
+        - name: S3_BUCKET
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: aws.s3.bucket
+        - name: S3_URL
+          value: "http://$S3_BUCKET.s3.amazonaws.com/"
```

Comment:
Is this a global S3 bucket used in other places, or is it dedicated especially for this app?